### PR TITLE
Malware fields

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MalwareInvestigationAndResponse/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Incident Fields
+##### New: **Malware Investigation Summary**
+- %%UPDATE_RN%% (Available from Cortex XSOAR 6.5.0).
```
Comment (ssokolovich) [inline Packs/MalwareInvestigationAndResponse/ReleaseNotes/1_0_1.md:4]: This should be removed ( or add a description at least).

File: Packs/MalwareInvestigationAndResponse/IncidentFields/incidentfield-Malware_Detailed_Investigation_Summary.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "locked": false,
+    "name": "Malware Detailed Investigation Summary",
+    "neverSetAsRequired": false,
+    "openEnded": false,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "shortText",
```
Comment (ssokolovich) [inline Packs/MalwareInvestigationAndResponse/IncidentFields/incidentfield-Malware_Detailed_Investigation_Summary.json:24]: Should be Multi-Select no? Or Am I missing here something?

Comment (David-BMS) [reply Packs/MalwareInvestigationAndResponse/IncidentFields/incidentfield-Malware_Detailed_Investigation_Summary.json:24]: No this is short text deliberately if you set the json to an array field it looks like 
 
image

File: Packs/MalwareInvestigationAndResponse/ReleaseNotes/1_0_1.md (hunk 1 above)
Comment (David-BMS) [reply Packs/MalwareInvestigationAndResponse/ReleaseNotes/1_0_1.md]: Done

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**